SwiftUIDZ/SwiftUIDZ/ClimateControlView/ClimateControlView.swift

```diff
@@ -0,0 +1,350 @@
+            }
+    }
+    private func onChangeBottomSheetOffset() {
+        DispatchQueue.main.async {
+            climateControlViewModel.bottomSheetCurrentOffsetY = bottomSheetGestureOffsetValue.height + climateControlViewModel.bottomSheetLastOffsetY
+        }
+    }
+    
+    private func makeClimateGroupView(index: Int) -> some View {
+        VStack {
+            
+            HStack(spacing: 10) {
+                Text(climateControlViewModel.settingsClimate[index].settingName)
+                    .font(.title)
+                    .bold()
+                    .foregroundColor(Color("buttonGr1"))
+                    .frame(width: 70)
+                //  .offset(x: -10)
+                
+                Button {
+                    climateControlViewModel.settingsClimate[index].isAction.toggle()
+                    climateControlViewModel.settingsIndex = index
+                    climateControlViewModel.isActive()
+                    
+                } label: {
+                    ZStack {
+                        Circle()
+                            .fill(LinearGradient(colors: [Color.white.opacity(0.15), Color.black.opacity(0.35)], startPoint: .top, endPoint: .bottomTrailing))
+                            .frame(width: 60)
+                            .overlay(Circle().stroke(LinearGradient(colors: [Color.black.opacity(0.6), Color.white.opacity(0.2)], startPoint: .topLeading, endPoint: .bottomTrailing), lineWidth: 1))
+                        
+                            .padding(.all, 0)
+                            .background(Circle().fill(Color("back2").opacity(0)))
+                        
+                            .shadow(color: Color("ligth").opacity(0.2), radius: 3, x: 4)
+                            .shadow(color: Color("dark").opacity(0.2), radius: 3)
+                        Image(systemName: climateControlViewModel.settingsClimate[index].settingsImageName)
+                            .foregroundColor(climateControlViewModel.settingsClimate[index].isAction ? Color("GradientBottom") : .white)
+                    }
+                }
+
+              
+                
+                LockerSlider(value: Binding(get: {
+                    if climateControlViewModel.settingsClimate[index].isAction {
+                     return   climateControlViewModel.climateSliderValues[index]
+                    } else {
+                     return   0.0
+                    }
+                }, set: { newValue in
+                    if climateControlViewModel.settingsClimate[index].isAction {
+                        climateControlViewModel.climateSliderValues[index] = newValue
+                    } else {
+                        climateControlViewModel.climateSliderValues[index] = 0.0
+                    }
+                }), in: 0...15, step: 1)
+                    .frame(width: 230, height: 50)
+            }
+        }
+        .padding(.vertical, 12)
+    }
+    
+    var body: some View {
```

Comment:
```suggestion
    var body: some View {
```
публичный вар же выше метода и private